generator/java/build.gradle

```diff
@@ -45,17 +46,21 @@ task generateClient(type: GradleBuild) {
         
         def clientOutputDirPerVersion = clientOutputDir + "/" + parsedApiName;
         def specPath = it.path;
-        def packageBase =  criteoApiVersion; 
-        if (packageBase.matches("[0-9].*"))
-             packageBase= "v" + packageBase;
-
-        packageBase = "${criteoPackage}.${criteoService}.${packageBase}".replace('.', nameSeparator).toString(); 
-        def packageVersion = "${criteoApiVersion}.${generatorVersion}.${getFormattedDate()}"; // example : 2021_10.1.211110
+        def versionCategory = criteoApiVersion == 'preview' ? 'preview' : 'stable'
```

Comment:
Maybe we could replace versionCategory by "isPreviewVersion"?

Reply:
Just a question, where does the criteoApiVersion variable come from?

Reply:
Done